ui/src/wizard/StorageAccountListStep.ts

```diff
@@ -139,12 +141,22 @@ export class StorageAccountListStep<T extends types.IStorageAccountWizardContext
             });
         }
 
-        if (hasFilteredAccounts && this._filters.learnMoreLink) {
+        if (hasFilteredAccountsBySku && this._filters.learnMoreLink) {
             picks.push({
-                label: localize('filtered', '$(info) Some storage accounts were filtered. Learn more...'),
-                description: '',
-                suppressPersistence: true,
-                data: this._filters.learnMoreLink
+                label: localize('hasFilteredAccountsBySku', '$(info) Some storage accounts were filtered because of their sku. Learn more...'),
+                onPicked: async () => {
+                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                    await openUrl(this._filters.learnMoreLink!);
+                },
+                data: undefined
+            });
+        }
+
+        if (hasFilteredAccountsByLocation && location) {
+            picks.push({
+                label: localize('hasFilteredAccountsByLocation', '$(warning) Only storage accounts in the selected region "{0}" are shown.', location.displayName),
```

Comment:
No strong preference but I think the "info" and "warning" icons look a little strange right next to each other. Maybe the info icon could suffice for both?

Reply:
I had thought that too, but I think it'd be hard for a user to parse that they're different messages if the icons were the same.  Especially since the content of the messages are similar.